content/_data/upgrades/2-277-1.adoc

```diff
@@ -0,0 +1,50 @@
+==== Plugin updates
+
+The configuration user interface transition from HTML `table` layout to HTML `div` layout has required several plugin updates.
+Jenkins administrators should update the installed plugins when they update to Jenkins 2.277.1.
+Some older plugin releases will cause the Jenkins "Configure System" page to fail to save.
+Other older plugin releases will report Java runtime exceptions until they are upgraded to the most recent release.
+
+Popular plugins that need to be updated after upgrading to Jenkins 2.277.1 include:
```

Comment:
Inconsistent uses of update/upgrade in this section.

Reply:
Fixed in c27269d615cf9c58d31cae4b3c8e6819c0b6a0a9